chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -96,23 +102,24 @@ object CBPurchase {
         }
     }
 
+    /**
+     * This method will provide all the purchases associated with the current account based on the [inActivePurchases] flag set.
+     * And the associated purchases can be synced with Chargebee.
+     *
+     * @param [context] Current activity context
+     * @param [inActivePurchases] False by default. if true, only active purchases restores and synced with Chargebee.
+     * @param [completionCallback] The listener will be called when restore purchase completes.
+     */
+    @JvmStatic
+    fun restorePurchases(context: Context, inActivePurchases: Boolean = false, completionCallback: RestorePurchaseCallback){
```

Comment:
Can we rename `inActivePurchases` to `includeInActiveProducts` / `includeInActivePurchases`?

Reply:
Updated with `includeInActivePurchases`